exercises/exercise3.js

```diff
@@ -1,12 +1,92 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
-    handler: function () {
-      return 'Good Evening to You!';
+    label: 'Valley Girl Math Star, seriously',
+    prompt: 'say hi to #{name}',
+    handler: function (params) {
+      return 'Like welcome to Valley Girl Math Star demo, ' + params.name + '!';
     },
+    isReply: false,
+    isCaseSensitive: false,
+    isListening: true,
+  },
+
+  {
+    label: 'celsius to farenheit',
+    prompt: 'Enter celsius, rad conversion to farenheit #{amount as number}',
+    handler: function (params) {
+      const temperature = Math.round((params.amount * 9) / 5 + 32);
+      return 'So very ' + temperature + '!';
+    },
+    isListening: true,
+  },
+
+  {
+    label: 'number squared',
+    prompt: 'Like enter number to be squared #{amount as number}',
+    handler: function (params) {
+      const squaredNum = params.amount * params.amount;
+      return 'AWESOME ' + squaredNum + '!';
+    },
+    isReply: false,
+    isCaseSensitive: false,
+    isListening: true,
+  },
+  // Write an interaction where one user asks the bot
+  // "What's the area of a circle with radius X?" where X is any number
+  // The chatbot will respond to the user with the answer.
+  //
+  // mychatbot> mychatbot What's the area of a circle with radius 5?
+  // mychatbot responds to user "The area is 78.5398...."
+  {
+    label: 'circle calc',
+    prompt: 'Enter a radius to see the area of a circle #{amount as number}',
+    handler: function (params) {
+      const area = Math.PI * (params.amount * params.amount);
+      return 'Excellent, Area = ' + Math.ceil(area);
+    },
+    isReply: false,
+    isCaseSensitive: false,
+    isListening: true,
+  },
+  {
+    label: 'square root',
+    prompt: 'As if, enter num and will give square root #{amount as number}',
+    handler: function (params) {
+      const sqRoot = Math.sqrt(params.amount);
+      return 'Rad ! it is ' + sqRoot;
```

Comment:
Great use of Math.sqrt() from JS Basics, and still enjoying these sassy responses!